### DISCUSSION TITLE
Select Range within Layouts

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#10737] [Feature Requests]
Recently, I have been doing a lot of bulk updates to clean up items in a `metrics` Collection and therefore need to select many (20+) items on the Layouts (Table)... but not ALL items on the page. The rows I've been selecting have all been sequential, so I always want to...

**Have a SHIFT keyboard shortcut that allows me to select all items between two previously selected items.** More specifically:

1. Select a single item (the index-a)
2. HOLD SHIFT and Click another item (index-b)
3. All items in between are selected

Edge-cases:
* If you skip step one (index-a), it should treat the first item in the list as index-a
* It should use the **last single selection** as the index-a. So if you select three random items, only the LAST selection will count as index-a
* The SHIFT-Click does not set index-a... so you can SHIFT-Click different times and it keeps adjusting (forwards or backwards)

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
